scripts/process_jartic_parallel.py

```
@@ -0,0 +1,320 @@
+import csv
+import logging
+from concurrent.futures import ProcessPoolExecutor, as_completed
+from multiprocessing import cpu_count
+from tqdm import tqdm
+import time
+from src.infrastructure.data_reference import ExternalDataManager
+
+
+logging.basicConfig(
+    level=logging.INFO,
+    format='%(asctime)s - %(levelname)s - %(message)s'
```

Ruff (0.12.2)

Trailing comma missing

Add trailing comma

(COM812)